oneview_redfish_toolkit/tests/blueprints/test_subscription.py

```diff
@@ -199,6 +199,42 @@ def test_add_subscription_invalid_destination(self, uuid_mockup):
         self.assertEqual(status.HTTP_400_BAD_REQUEST, response.status_code)
         self.assertEqual("application/json", response.mimetype)
 
+    @mock.patch('uuid.uuid1')
+    def test_remove_subscription(self, uuid_mockup):
```

Comment:
can you check if the subscription still exists after deleting? Maybe if you use on get_subscription...

Reply:
Checked! thanks...

Reply:
I also added a new test to check the creation of a subscription with duplicated elements...